python/perf-kernels/streamk/streamk_kernel.py

```diff
@@ -0,0 +1,221 @@
+import triton
+import triton.language as tl
+
+
+@triton.jit()
+def get_new_pid(current_pid, num_sms):
+    # Number of XCDs
+    num_xcds = 8
+    # Number of pids per XCD in the new arrangement
+    pids_per_xcd = num_sms // num_xcds
```

Comment:
I thought the grid can have multiple of num_cus pids.

Reply:
For persistent kernel, grid has to be either num_cus or total_tiles if total_tiles < num_cus